daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -405,6 +405,25 @@ class RazerBlackWidowUltimate2013(_MacroKeyboard):
     }
 
 
+class RazerBlackWidowTournamentEdition2014(_MacroKeyboard):
+    """
+    Class for the Razer BlackWidow Tournament Edition 2014
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*Razer_Razer_BlackWidow_Tournament_Edition-(-if01)?-event-kbd')
```

Comment:
There is a typo here (extra -), the device is called " /dev/input/by-id/usb-Razer_Razer_BlackWidow_Tournament_Edition-if01-event-kbd -> ../event19"

so it should be Tournament_Edition(-if01)

Reply:
Sorry for the delay, I've fixed it. Could you please test with the most recent version again?